frontend/src/libs/enums/sidebar-tabs.enum.ts

```diff
@@ -0,0 +1,7 @@
+const TabNames = {
```

Comment:
TabName. enum name should be singular

Reply:
I am using TabName for single tab type. Changed TabNames to TabsName, is that okay?